content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Istio CNI Install Options
+description: Instructions for installing and using Istio with the Istio CNI plugin.
+weight: 70
+keywords: [kubernetes,cni]
+---
+
+The Istio CNI plugin implements the networking details for injecting the Istio sidecar proxy between application pods
```

Comment:
Not a big fan of "details" here.  Would "connections" or links/linkages  work better.

Reply:
maybe just remove "details" and leave it at "networking"?

Reply:
sure that works